Simulator/simulator/garden.py

```diff
@@ -386,8 +389,8 @@ def distribute_water(self):
 
                     plant_types_and_ids.pop(i)
 
-            # Water evaporation/drainage from soil
-            point['water'] = max(0, point['water'] - self.drainage_rate)
+            # Water evaporation per square cm (grid point)
+            point['water'] = max(0, point['water'] - 0.01*0.01*self.evaporation_rate)
```

Comment:
Is 0.01*0.01 from your tuning experiments?

Reply:
Evaporation rate is per square meter - point is square centimeter.

Reply:
Does this conflict with line 4 and 26 of plant_presets.py where inches are used?